lib/chef/formatters/base.rb

```diff
@@ -22,6 +22,7 @@
 require "chef/formatters/error_description"
 require "chef/formatters/error_mapper"
 require "chef/formatters/indentable_output_stream"
+require "chef/dist"
```

Comment:
should really move that down a level so that this line appears in every file that references Chef::Dist -- otherwise either someone will delete it here, or someone will add it there...

Reply:
I had a doubt to set it in base or in each file, updating to set it in relevant file then.